### PR TITLE
Clean code to prepare for changes

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Cleaning up the style and removing contracts and unnecessary attributes to prepare for my pending API fill-in changes. No functional changes.

@weshaggard, @danmosemsft

### CODE REVIEW

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs [hunk 1]
```diff
@@ -5,61 +5,66 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
```
Comment (stephentoub) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: Do we need to change the ref file like this?

**Comment by stephentoub** [comment]
Changes to the src files LGTM.  We've not done any such cleanup in the ref files, though, and I'm not sure we want to start here (though I don't have a fundamental problem with it)... @weshaggard?

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs (hunk 1 above)
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: I'm about to make a big change to this file, wanted it to be consistent.

**Comment by JeremyKuhne** [comment]
> Changes to the src files LGTM. We've not done any such cleanup in the ref files, though, and I'm not sure we want to start here (though I don't have a fundamental problem with it)... @weshaggard?

I'm only doing it because I have a lot of changes to make- otherwise I'd leave it alone.

**Comment by weshaggard** [comment]
I would prefer not to change the formatting of the ref .cs file as I want it to match the output of genapi. So instead of doing the manual formatting I would prefer we just run genapi against the implementation.

**Comment by JeremyKuhne** [comment]
> I would prefer not to change the formatting of the ref .cs file as I want it to match the output of genapi. So instead of doing the manual formatting I would prefer we just run genapi against the implementation.

Ok. Do we have a good place to document this if it isn't already?

**Comment by stephentoub** [comment]
If you've removed all uses of contracts-related stuff, can it be removed from the project.json as well?

**Comment by JeremyKuhne** [comment]
> If you've removed all uses of contracts-related stuff, can it be removed from the project.json as well?

Yes, I plan to with the actual code changes.

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**